### PR TITLE
added layout for capris page

### PR DESCRIPTION
**Pull request by atphy**
- Reduced margin around navbar logo image, which was making the bar a bit thicker than needed
- Set up most layout aspects for the Buy Capris page

TO-DO:
- Align price on same line as size dropdown (I stared at it too long and can't get it to work right now)

### CODE REVIEW

File: capris.html [hunk 1]
```diff
@@ -21,8 +21,27 @@
...
   </header>
-  <h1>Buy Capris Page</h1>
-
+  <div id="caprismain">
+    <img id="buycaprispic" src="https://via.placeholder.com/400x500">
+    <div id="nameandrating">
+      <h5 id="buycaprisname">Placeholder Capris Name</h5><h2>☆☆☆☆☆</h2>
+    </div>
+    <div class="caprisinfobox">
+      <p id="#caprisinfo">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
```
Comment (RyanBeiden) [inline capris.html:30]: This may need the ```#``` removed.

Comment (ryanbeiden) [inline capris.html]: This may need the ```#``` removed.

**Review by ryanbeiden — CHANGES_REQUESTED** [review]
Just that extra ```#``` and then it looks great and works for me!

File: capris.html (hunk 1 above)
Comment (atphy) [reply capris.html]: Good catch, updated!

**Review by ryanbeiden — APPROVED** [review]
Love it 👍

**Pull request merged**